contrib/src/test/resources/RegexSplitterschema.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Does this operator need a CSV like schema definition ? How is this operator using a separator and quouteChar for parsing inputs ?